float64 vx
float64 vy
float64 vz
float64 fom
float64 altitude
waterlinked_dvl/Transducer[] transducers
bool velocity_valid
int32 status
string format
float64 time